tests/integration_tests/asn1_choice/resources/taste/asn1_choice.msc

```diff
@@ -0,0 +1,12 @@
+mscdocument asn1_choice /* MSC AND */;
```

Comment:
Is this needed?

Reply:
This applies to all instances in tests

Reply:
It isn't required, I'll remove msc files.

Reply:
Done.